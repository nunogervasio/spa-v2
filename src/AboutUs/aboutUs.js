import React from "react";
import styled from "styled-components";
import { Card, Section, Container } from "../components";

// const AboutUs = () => {
const AboutUs = () => {
  return (
    <Section>
      <Container>
        <Card />
        <Card />
        <Card />
      </Container>
    </Section>
  );
};

export default AboutUs;
