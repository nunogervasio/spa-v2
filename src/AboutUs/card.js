import React from "react";
import styled from "styled-components";
import { Button } from "./button";
import { Paragraph } from "./paragraph";
import { Image } from "./image";
import { Container } from "./container";
import { Title } from "./title";
import yogi from "../images/yoga-guy.jpg";

const CardContainer = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  flex-grow: 1;
  max-width: 1100px;
  margin: 0.75em;
  padding: 0.5em;
  background-color: white;
  border-radius: 2%;
`;

const Card = () => {
  return (
    <CardContainer>
      {/* <Image src="https://via.placeholder.com/260" /> */}
      <Image src={yogi} />
      <Title>Default Title</Title>
      <Paragraph>
        Ofrecemos diferentes tipos de yoga, power ashtanga, vinyasa flow,
        vinyasa chakra, yin y pilates. Con m√°s de 20 clases por semana y una
        variedad de instructores certificados, nuestro horario y clases son para
        estudiantes de todos los niveles.
      </Paragraph>
      <Button>Normal</Button>
      {/* <Button primary>Normal Plus</Button> */}
    </CardContainer>
  );
};

export default Card;
