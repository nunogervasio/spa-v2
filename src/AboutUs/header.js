import React from "react";
import styled from "styled-components";
import { Title } from "./title";

// export const Header = styled.header`
//   border: 2px solid palevioletred;
// `;

const SubTitle = styled(Title)`
  font-size: 0.9em;
`;
const Header = () => {
  return (
    <header>
      <Title>Header of Page</Title>
      <SubTitle>Subtitle of the page...</SubTitle>
    </header>
  );
};

export default Header;
