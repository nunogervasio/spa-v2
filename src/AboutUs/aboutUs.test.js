import React from "react";
import { shallow } from "enzyme";
import styled from "styled-components";
import { enzymeFind } from "styled-components/test-utils";
import AboutUs from "./aboutUs";
import toJson from "enzyme-to-json";

const wrapper = shallow(<AboutUs />); // StyledButton is your styled-components's component
// const renderedComponent = enzymeFind(wrapper, AboutUs);

it("renders properly", () => {
  expect(toJson(wrapper)).toMatchSnapshot();
});
// expect(renderedComponent.prop('className')).toBeDefined();
