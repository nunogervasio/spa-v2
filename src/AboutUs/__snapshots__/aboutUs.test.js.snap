// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<Styled(styled.section)>
  <Styled(styled.div)>
    <styled.h1>
      About Us
    </styled.h1>
    <Styled(styled.h1)>
      What We Provide The Best
    </Styled(styled.h1)>
    <styled.p>
      sed do eiusmod tempor incididunt ut Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet, eiusmod tempor incididunt ut labore consectetur adipiscing sed do eiusmod tempor incididunt ut Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet consectetur adipiscing sed do eiusmod tempor incididunt ut Lorem ipsum dolor sit amet
    </styled.p>
    <styled.button>
      Read More
    </styled.button>
  </Styled(styled.div)>
  <Styled(styled.div)>
    <styled.div>
      <Styled(styled.img)
        src="delivery-truck.svg"
      />
      <styled.h1>
        Land Transport
      </styled.h1>
    </styled.div>
    <styled.div>
      <Styled(styled.img)
        src="delivery-truck.svg"
      />
      <styled.h1>
        Land Transport
      </styled.h1>
    </styled.div>
    <styled.div>
      <Styled(styled.img)
        src="delivery-truck.svg"
      />
      <styled.h1>
        Land Transport
      </styled.h1>
    </styled.div>
    <styled.div>
      <Styled(styled.img)
        src="delivery-truck.svg"
      />
      <styled.h1>
        Land Transport
      </styled.h1>
    </styled.div>
  </Styled(styled.div)>
</Styled(styled.section)>
`;
